projects/ellipsis/index.js

```diff
@@ -0,0 +1,135 @@
+/*==================================================
```

Comment:
This is a nitpick, but there's no need for these huge comments, I usually just remove them myself